infrastructure/monitoring/README.md

```diff
@@ -1,312 +1,23 @@
-<!-- markdown-link-check-disable-next-line -->
-The following guide is an adapted version of this [blog post](https://computingforgeeks.com/how-to-monitor-bind-dns-server-with-prometheus-and-grafana/).
-
-#### Bind configuration
-1. Download the latest release of the `bind_exporter` binary:
-    ```bash
-    $ wget -qO - https://api.github.com/repos/prometheus-community/bind_exporter/releases/latest | grep browser_download_url | grep linux-amd64 |  cut -d '"' -f 4 | wget -qi -
-    ```
-2. Extract the binary and move it to the /url/local/bin folder:
-    ```bash
-    $ sudo tar xvf bind_exporter-*.tar.gz --directory /usr/local/bin --wildcards --strip-components 1 '*/bind_exporter'
-    ```
-3. Edit `/etc/bind/named.conf.options`, and open a statistics channel:
-    ```
-    statistics-channels {
-        inet 127.0.0.1 port 8053 allow { 127.0.0.1; };
-    };
-    ```
-4. Reload the `bind9` configuration:
-    ```bash
-    $ sudo rndc reload
-    ```
-5. Add the `prometheus` system user account
-    ```bash
-    $ sudo groupadd --system prometheus
-    $ sudo useradd -s /sbin/nologin --system -g prometheus prometheus
-    ```
-6. Create a `systemd` unit file for `bind_exporter`:
-    ```bash
-    $ sudo tee /etc/systemd/system/bind_exporter.service<<'EOF'
-    [Unit]
-    Description=Prometheus
-    Documentation=https://github.com/digitalocean/bind_exporter
-    Wants=network-online.target
-    After=network-online.target
-
-    [Service]
-    Type=simple
-    User=prometheus
-    Group=prometheus
-    ExecReload=/bin/kill -HUP $MAINPID
-    ExecStart=/usr/local/bin/bind_exporter \
-      --bind.pid-file=/var/run/named/named.pid \
-      --bind.timeout=20s \
-      --web.listen-address=0.0.0.0:9153 \
-      --web.telemetry-path=/metrics \
-      --bind.stats-url=http://localhost:8053/ \
-      --bind.stats-groups=server,view,tasks
-
-    SyslogIdentifier=prometheus
-    Restart=always
-
-    [Install]
-    WantedBy=multi-user.target
-    EOF
-    ```
-7. Reload `systemd` and start the `bind_exporter` service:
-    ```bash
-    $ sudo systemctl daemon-reload
-    $ sudo systemctl enable bind_exporter.service
-    $ sudo systemctl restart bind_exporter.service
-    ```
-8. Configure `iptables` to limit the access to `bind_exporter` to specific IP addresses (optional):
-    ```bash
-    $ sudo iptables -A INPUT -p tcp -s 130.192.0.0/16 --dport 9153 -j ACCEPT
-    $ sudo iptables -A INPUT -p tcp -s 192.168.0.0/16 --dport 9153 -j ACCEPT
-    $ sudo iptables -A INPUT -p tcp --dport 9153 -j DROP
-    ```
+## Table of contents
+- [kube-prometheus-stack](./kube-prometheus-stack)
+  - [kube-prometheus-stack-values.yaml](./kube-prometheus-stack/kube-prometheus-stack-values.yaml) 
+  - [Thanos](./kube-prometheus-stack/thanos)
+    - [thanos-values.yaml](./kube-prometheus-stack/thanos/thanos-values.yaml)
+    - [manifests](./kube-prometheus-stack/thanos/manifests)
```

Comment:
Would personally remove this table of contents, as already presented in the description above.